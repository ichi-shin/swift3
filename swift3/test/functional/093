#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-5-2_6

. ./common

echo '<VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
   <Status>Enabled</Status>
</VersioningConfiguration>' > $tmp.versioning.xml

S3USER=tester
_s3_put /hoge
_s3_put /hoge?versioning -T $tmp.versioning.xml

S3USER=tester2
_s3_get /hoge?versioning -D - | _filter_curl xml

