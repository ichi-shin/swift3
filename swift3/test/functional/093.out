QA output created by 093
> s3curl --id tester -- -X PUT http://SWIFT_HOST/hoge... 200
> s3curl --id tester -- -X PUT -T /TMP.versioning.xml http://SWIFT_HOST/hoge?versioning... 200
> s3curl --id tester2 -- -X GET -D - http://SWIFT_HOST/hoge?versioning... 403
HTTP/1.1 403 Forbidden
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>AccessDenied</Code>
  <Message>Access Denied.</Message>
  <RequestId>TXID</RequestId>
</Error>
